libraries/typings-generator/src/TypingsGenerator.ts

```diff
@@ -34,6 +34,8 @@ export interface ITypingsGeneratorOptions<TTypingsResult = string | undefined> {
  * @public
  */
 export class TypingsGenerator {
+  private _targetMap: Map<string, Map<string, boolean>>;
```

Comment:
Provide a comment showing what the map is used for ex.
```ts
// target file path -> Map<dependency, exists>
```

Reply:
Added comments. Thanks.